### DISCUSSION TITLE
Basic functionality of a component should always work

### DISCUSSION THREAD

**Discussion by rktaxali** [vuetifyjs/vuetify#20816] [General] [answered]
First of all, I want to thank the developers and community who put special efforts to release Vuetify components for public. 

I have been using Vuetify components for over two years. While the basic components, like buttons, text boxes, select boxes, etc. work flawlessly, the advanced components often have bugs that sometimes, even the basic functionality does not work. And, the situation becomes confusing for a developer (like me) who feels that the component is not working because I am not using it correctly. For example, last week, I tried to use `<v-calendar>`, and I noticed that Next and Prev buttons don't emit the event, though the [documentation](https://vuetifyjs.com/en/api/v-calendar/), clearly lists `next` and `prev` events. It took me quite sometime to realize that these event don't work. Next, I was using my calendar in the 'week' mode and the event slot was not working. Again, it took me sometime to figure out that though the event slot works in the **month** mode, it does not work in **week** mode!

In my opinion, the functioning of the Next and Previous buttons and the event slot are the basic needs of a calendar, and they should always work. Perhaps, for each component, someone may check these before releasing/upgrading a component to public. And, in case bugs are posted for non working of the basic features, these may be fixed on priority.

**Comment by WilcoSp** [comment, marked as answer]
The v-calendar components are in labs and are thus not finished yet, they're made available so they could be tested and to be able for feedback/bug reports to be given. Labs components shouldn't be used in production.

more info: https://vuetifyjs.com/en/labs/introduction/#what-is-labs
